api-client/src/mockData/cssQuiz/cssQuestions.ts

```diff
@@ -0,0 +1,148 @@
+import {QuestionJsonldQuestionRead} from "../../generated/openapi"
+
+const CSSQuestions: QuestionJsonldQuestionRead[] = [
+    {
+        '@context': "/api/v1/contexts/Question",
+        '@id': "/api/v1/question/15",
```

Comment:
Hello @pfwd, got small confusion related ids on this points, should we keep them unique?

Reply:
Yes they should be unique. Ideally they should match those in the database but I'm not sure that will be possible once we start adding more.

Reply:
Perhaps we need a script that will re-write the IDs.  I will need to give it some thought 🤔 
For now, please keep them as unique as you can.

Reply:
@pfwd ok thank you, I am just opening a request for this one we can handle it in an another issue maybe, it's hard with static values at this point